docker-compose/obam-with-obkm/obam/au/repository/deployment/server/jaggeryapps/store/site/conf/site.json

```diff
@@ -1,92 +0,0 @@
-{
```

Comment:
Why is this and other config files deleted?

Reply:
Since we have removed store in OB 2.0 added the new configs for devportal.